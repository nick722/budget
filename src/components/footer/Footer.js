import React from "react";

import "./footer.scss";

const Footer = () => {
  return <footer>&#169; Copyright 2019</footer>;
};

export default Footer;
