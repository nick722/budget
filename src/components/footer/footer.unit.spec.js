import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

describe("Footer component", () => {
  it("should render into the document", () => {
    const component = shallow(<Footer />);
    expect(component).toHaveLength(1);
  });
});
