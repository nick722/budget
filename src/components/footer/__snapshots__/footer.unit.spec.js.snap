// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render into the document 1`] = `
<footer>
  Â© Copyright 2019
</footer>
`;
